### DISCUSSION TITLE
`biome lint --apply-unsafe=ruleId`

### DISCUSSION THREAD

**Discussion by unional** [biomejs/biome#2223] [Ideas]
This allow existing repository gradually transition to `biome` and separate the fix into separate commits.

**Comment by Conaclos** [comment]
Could #58 (and potentially #57) fulfill your needs?

**Comment by ematipico** [comment]
Closing, because that's already planned (if anyone wants to help, you're welcome) in https://github.com/biomejs/biome/issues/58
